service/src/main/java/uk/gov/hmcts/reform/fpl/service/TaskListRenderer.java

```diff
@@ -38,9 +38,28 @@
 public class TaskListRenderer {
 
     private final String imagesBaseUrl;
+    private final FeatureToggleService featureToggleService;
 
-    public TaskListRenderer(@Value("${resources.images.baseUrl}") String imagesBaseUrl) {
+    public TaskListRenderer(
+            @Value("${resources.images.baseUrl}") String imagesBaseUrl,
+            FeatureToggleService featureToggleService) {
         this.imagesBaseUrl = imagesBaseUrl;
+        this.featureToggleService = featureToggleService;
+    }
+
+    //TODO consider templating solution like mustache
```

Comment:
Are we doing anything with this TODO @tomaszpowroznik? Is this a tech debt, spike etc?

Reply:
afaik, we need approval from architects, Adam already started some discussion with Khaled